samples/gcp-pubsub-source/dumper/dumper.go

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Optional: does it make sense to elevate this to `eventing-sources/cmd`? I needed it a couple days ago while testing KES.

Reply:
Also optional: I might run this as a standard Deployment and Service instead of a knative Service to eliminate as many variables as possible. While debugging today we had some uncertainty about where the issue was, and knative Service is complex enough that we had to experimentally eliminate it as a cause. Plus, using a Service requires the user to install a lot of extra stuff for no obvious benefit in this demo.

Counter argument: Making this a knative Service is a good intro to the cool features of knative serving.

Our bug was actually in eventing not serving, so we should have suspected our own code first. :smile:

Reply:
+1 to making it into Knative Service.

Reply:
Let's do the move in a follow up PR. At the same time I'll create two YAMLs, one for a Deployment, the other for a Knative Service.